backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -93,23 +95,36 @@ def resolve_fetch_failures_0_members(
         # Fetch data
         csets_and_members: Dict[str, List[Dict]] = fetch_cset_and_member_objects(
             codeset_ids=list(failed_cset_ids), flag_issues=False)
-        # - filter success & track results
-        csets_and_members['OMOPConceptSet'] = [
-            x for x in csets_and_members['OMOPConceptSet'] if x['member_items']]
-        cset_is_draft_map.update({x['properties']['codesetId']: x['properties']['isDraft'] for x in csets_and_members['OMOPConceptSet']})
-        success_cases: List[int] = [
-            x['properties']['codesetId'] for x in csets_and_members['OMOPConceptSet']]
-        failed_cset_ids = list(set(failed_cset_ids) - set(success_cases))
+
+        # - identify discarded drafts
+        # TODO: ascertain discarded drafts
+        #  - how to ascertain?
+        #    - 1 (ok~): detect discarded drafts as not returned from fetch_cset_and_member_objects(). ~confirm w/ amin et al
+        #    - 2 (x): Ask Amin et al: is the enclave API reliable for discarded drafts? Some of these are appearing in the
+        #    enclave API but not in the UI. x: I think (2) is not a case I need to worry about. i think they weren't discared.
+        #    - This line below should be enough to handle. Just want to verify (1)
+        discarded_cset_ids: Set[int] = set([x for x in failed_cset_ids if x not in csets_and_members['OMOPConceptSet']])
+        # TODO: mark discarded drafts resolved
```

Comment:
## FYI: Investigation
FYI, in this run that I looked at (new data from doing a refresh since Jan 1st), there is:

20 fetch failures
- 1 empty return (probably discarded draft)
- 19 returned from `fetch_csets_and_members()`
  - 2 success cases (members fetched)
  - 17 persistent drafts (appear in TermHub & in enclave API but not in enclave UI)